public/globalRanking.php

```diff
@@ -285,7 +285,7 @@
                 echo "<td class='text-right'>" . localized_number($dataPoint['Points']) . "</td>";
             }
 
-            echo "<td class='text-right'>" . localized_number($dataPoint['TotalAwards']) . "</td></tr>";
+            echo "<td class='text-right'>" . localized_number($dataPoint['TotalAwards'] ?? 0) . "</td></tr>";
```

Comment:
We don't show a number when it's 0 in some areas. 
We could adjust localized_number to accept null to then return an empty string for convenience.

Reply:
In this case, I think saying the user has "0 "masteries is more appropriate than leaving the column blank.